docs/UserGuide.adoc

```diff
@@ -214,25 +232,27 @@ This command allows you to add ingredients to a recipe.
 |===
 | Parameters | Result
 
-|`recipe 1 add ingredient i/Egg q/2`
+|`recipe 1 add ingredient i/Eggs q/2`
 |Adds 2 eggs to the 1st recipe's list of ingredients.
 
 |`recipe 2 add i/Milk ingredient q/200 ml`
 |Adds 200 ml of milk to the 2nd recipe's list of ingredients.
 |===
 
 ==== Remove an ingredient from a recipe
-You can remove ingredients that spoil the taste of the dish using this command.
-If the quantity specified is of smaller value, the ingredient quantity will be subtracted and updated.
-Please ensure that the unit specified corresponds accordingly.
+This command allows you to remove ingredients that spoil the taste of a dish from a recipe.
+You can indicate the quantity you want to remove for an ingredient, which should not be greater that than the
+ingredient's quantity.
+Please ensure that the unit of the quantity matches the ingredient's unit in the recipe.
+If you do not indicate any quantity, all of the specified ingredient will be removed from the recipe.
```

Comment:
Perhaps it can be better reflected that if there are multiple ingredients with the same ingredient name but different quantity, then in that case, providing no quantity will remove the ingredient for all kinds of quantities.

Reply:
I have updated the UG accordingly. Do you think it is clear enough now?